docs/es/cookbook/components/index.md

```diff
@@ -1,9 +1,9 @@
 # Componentes
```

Comment:
Well I meant the whole page :) Seems to be the only one though

Reply:
i already pushed the fix for that 😄

Reply:
Well AFAIK you only rollbacked the first line of the file 😬